src/coreclr/nativeaot/Runtime/inc/rhbinder.h

```diff
@@ -28,6 +28,11 @@ struct DispatchCellInfo
     uint8_t HasCache = 0;
     uint32_t MetadataToken = 0;
     uint32_t VTableOffset = 0;
+
+    uint32_t GetVTableOffset() const
+    {
+        return VTableOffset;
+    }
```

Comment:
Why is this needed if all the fields are public?

Reply:
The VTableOffset field doesn't exist in the CoreCLR variant of this type.